js/web/lib/wasm/binding/ort-wasm-simd.d.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export interface OrtWasmSimdModule extends EmscriptenModule {
```

Comment:
is this interface shares same definition to OrtWasmModule? if so, we don't need to define it twice; instead OrtWasmSimdModule can extends from OrtWasmModule

Reply:
yes, let me it